client/landing/gutenboarding/onboarding-block/style-preview/style.scss

```
@@ -40,17 +41,41 @@
 
 .style-preview__preview {
 	grid-area: preview;
-
-	border: 1px solid var( --studio-gray-5 );
-	box-shadow: 0 4px 14px rgba( 0, 0, 0, 0.25 );
+	margin: 0 auto;
+	background: var( --studio-white );
+	box-shadow: 0 0 0 1px var( --studio-gray-5 ), 0 4px 14px rgba( 0, 0, 0, 0.25 );
 	border-radius: 4px;
 	overflow: hidden;
+	width: 100%;
+
+	&.is-viewport-mobile,
+	&.is-viewport-tablet {
+		width: 375px;
+		height: 812px;
```

Yes, thanks! I'll grab the mobile dimensions from below:

```suggestion
		// Mobile dimensions, tablet is overridden below
		width: 351px;
		height: 690px;
```